packages/grafana-ui/src/components/Table/TableCellInspector.tsx

```diff
@@ -2,16 +2,16 @@ import { isString } from 'lodash';
 import React from 'react';
 
 import { ClipboardButton } from '../ClipboardButton/ClipboardButton';
-import { Modal } from '../Modal/Modal';
+import { Drawer } from '../Drawer/Drawer';
 import { CodeEditor } from '../Monaco/CodeEditor';
 
-interface TableCellInspectModalProps {
+interface TableCellInspectorProps {
   value: any;
   onDismiss: () => void;
   mode: 'code' | 'text';
 }
 
-export function TableCellInspectModal({ value, onDismiss, mode }: TableCellInspectModalProps) {
+export function TableCellInspector({ value, onDismiss, mode }: TableCellInspectorProps) {
```

Comment:
If we are renaming the exported function I think we also will need to modify everywhere else in the code this file is used / referenced 🤔